Update debug.md - remove logger level debug, add note about monitorin… (#5365)
OttoWinter/esphomedocs: content/components/debug.md

The `debug` component can be used to debug problems with ESPHome. At startup, it prints
a bunch of useful information like reset reason, free heap size, ESPHome version and so on.
It also allows you to get the same information as a text sensor, and to monitor the state of the

> ESP heap memory (free space, maximum free block size and fragmentation level) and the main-loop timing. Monitoring can be done in the terminal or web server component if it has been configured.

Reviewer: ```suggestion
ESP heap memory (free space, maximum free block size and fragmentation level) and the main-loop timing. Monitoring can be done via the logger or web server component if it has been configured.
```
